fhir-server-test/src/test/resources/nats/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: "3.7"
+networks:
+  main:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.19.0.0/24
+      driver: default
+services:
+    nats-node1:
+        image: nats-streaming:0.17.0
+        container_name: nats-node1
+        volumes:
+            - .:/nats
+        command: --config /nats/config-1.conf
+        networks:
+            - main
+        ports:
+            - 4222:4222
+            - 6222:6222
+    nats-node2:
+        image: nats-streaming:0.17.0
+        container_name: nats-node2
+        volumes:
+            - .:/nats
+        command: --config /nats/config-2.conf
+        networks:
+            - main
+        ports:
+            - 4223:4222
+            - 6223:6222
+    nats-node3:
+        image: nats-streaming:0.17.0
+        container_name: nats-node3
+        volumes:
+            - .:/nats
+        command: --config /nats/config-3.conf
+        networks:
+            - main
+        ports:
+            - 4224:4222
+            - 6224:6222
```

Comment:
```suggestion
            - 4224:4224
```

Reply:
I think its actually listening on 4224 and not 4222

Reply:
The external port is 4224, but the internal port is 4222, so 4224:4222 is correct.

Reply:
```
nats-node3     | [1] 2020/04/24 21:39:41.002469 [INF] Listening for client connections on 0.0.0.0:4224
nats-node3     | [1] 2020/04/24 21:39:41.002510 [INF] Server id is NCPFPS637V2OS77UF453BC7VRAKBHHA7BBCXE3DV3Q7ISDQGEEA3QJYB
nats-node3     | [1] 2020/04/24 21:39:41.002524 [INF] Server is ready
```